### PR TITLE
perf: Use env vars to configure Langfuse Client

### PR DESCRIPTION
**Pull request by khareyash05**
## Problem

Added env vars support for Langfuse Client
Fixes [https://github.com/langfuse/langfuse/issues/232](https://github.com/langfuse/langfuse/issues/232)

### Bump level

- [ ] Major
- [x] Minor
- [ ] Patch

### Libraries affected

- [ ] All of them
- [x] langfuse
- [ ] langfuse-node

### Changelog notes

- Added support for env vars

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,24 +18,28 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18, 20]
+        node-version: [16, 18, 20]
```
Comment (ksjogo) [inline .github/workflows/ci.yml:21]: Support for 16 got dropped. Not sure if the change was intentional.

Comment (khareyash05) [reply .github/workflows/ci.yml:21]: Oops while installing packages, I think it got omitted because 16 LTS is now out of support. Will add it !

Comment (ksjogo) [reply .github/workflows/ci.yml:21]: Well, actually the other way round. Should kept being removed. ;)

Comment (khareyash05) [reply .github/workflows/ci.yml:21]: I am not sure why it happened .Will do that!

**Comment by marcklingen** [comment]
I'm sorry for not merging this early on, @maxdeichmann included this change when doing a refactor of the constructor logic. we appreciate your work on this!

**Pull request closed without merging**